addons/auth_signup/tests/test_reset.py

```diff
@@ -0,0 +1,19 @@
+# rerun TestWebLogin tests with auth_signup installed
+from odoo.tests.common import HttpCase
+
+class TestResetPassword(HttpCase):
+    def test_password(self):
```

Comment:
test_reset_password ?

Would also add a void line between class and test definitions :)